docs/examples/nested_regions.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region, Page
+from selenium.webdriver.common.by import By
+
+
+class MainPage(Page):
+
+    @property
+    def menu1(self):
+        root = self.find_element(By.ID, "menu1")
+        return Menu(self, root=root)
+
+    @property
+    def menu2(self):
+        root = self.find_element(By.ID, "menu2")
+        return Menu(self, root=root)
+
+
+class Menu(Region):
+
+    @property
+    def entries(self):
+        items = self.find_elements(*Entry._root_locator)
+        return [Entry(self.page, item) for item in items]
```

Comment:
I prefer to put this all on one line:

```python
return [Entry(self.page, item) for el in self.find_elements(*self._entries_locator)]
```

When I do define a variable for the elements I tend to name it `els`:

```python
els = self.find_elements(*self._entries_locator)
return [Entry(self.page, item) for el in els]
```

Reply:
The one-liner absolutely makes sense. I did split the list comprehension into two because I am still struggling with the `_root_locator` which is a "private" attribute here that shouldn't be accessed from outer scope anyway.